### PR TITLE
Release/r104

### PR DESCRIPTION
**Pull request by chuwy**
- [x] Rebase
- [x] CHANGELOG
- [x] Date in CHANGELOG

### CODE REVIEW

**Comment by chuwy** [comment]
Rebased on top of master.

File: 3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb [hunk 1]
```diff
@@ -444,15 +447,20 @@ def initialize(debug, staging, enrich, staging_stream_enrich, shred, es, archive
...
               "--src"        , enrich_final_output, # Opposite way round to normal
               "--dest"       , ENRICH_STEP_OUTPUT,
               "--srcPattern" , src_pattern,
               "--s3Endpoint" , s3_endpoint
-            ] # Either user doesn't want compression, or files are already compressed
+            ]
+
+            if stream_enrich_mode then
+              copy_to_hdfs_step.arguments << "--outputCodec" << "none"
+            end
```
Comment (BenFradet) [inline 3-enrich/emr-etl-runner/lib/snowplow-emr-etl-runner/emr_job.rb]: i think we need `"--outputCodec" << "none"` in both modes.

File: 3-enrich/emr-etl-runner/spec/snowplow-emr-etl-runner/cli_spec.rb [hunk 2]
```diff
@@ -239,9 +239,22 @@ def resource(name)
...
+      expect {
+        Cli.validate_and_coalesce(a, s)
+      }.to raise_exception( ConfigError, "cannot skip staging nor enrich in stream enrich mode. Either skip staging_stream_enrich or resume from shred")
+    end
+
+    it 'should reject --resume-from enrich in stream enrich mode' do
+      a[:resume_from] = 'enrich'
+      expect {
+        Cli.validate_and_coalesce(a, s)
+      }.to raise_exception( ConfigError, "cannot resume from enrich in stream enrich mode")
```
Comment (BenFradet) [inline 3-enrich/emr-etl-runner/spec/snowplow-emr-etl-runner/cli_spec.rb]: you're missing a couple of tests for archive_raw and skip enrich also nit: `to raise_exception(Config...`

**Comment by chuwy** [comment]
Thanks @BenFradet! I will address feedback re `outputCompression` and do some more tests.

**Comment by chuwy** [comment]
Hey @BenFradet! This is ready from me.

**Comment by BenFradet** [comment]
merging when green

**Pull request merged**